README.md

```diff
@@ -32,18 +32,19 @@ USERNAME
 PASSWORD
 ```
 
-7. Run `docker-compose build`
+7. Run `sudo docker-compose  -f docker-compose.yaml -f docker-compose.prod.yaml build`
 8. Run backend build steps in a venv
-9. Run `docker-compose up`
+9. Run `sudo docker-compose  -f docker-compose.yaml -f docker-compose.prod.yaml up`
 
 #### To reset your database
 
-Run `./scripts/reset-database.sh` (Linux) or `./scripts/reset-database.ps1` (Windows)
+1. Run `cd scripts` 
```

Comment:
Running from project folder was already correct: the scripts will run `docker-compose` in the current / root folder, which is also where docker-compose.yaml is. I do not know the semantics of how docker-compose behaves if you run it in a child folder.

Reply:
Ok, I roll back this one.
But, in that case, would not be clear that all the script `sandbox.py` in "scripts" would run correctly also from the root folder?

I think that was the reason of our confusion.

Reply:
sandbox.py would, I believe, not run outside the script folder, as it opens files relative to current folder